repo-scripts/size-analysis/bundle-definitions/database.json

```diff
@@ -0,0 +1,256 @@
+[
+  {
+    "name": "Read data once",
```

Comment:
@schmidt-sebastian  Can you please take a look at the Database/Firestore/Storage use cases to see if you want to add anything, or if anything is inaccurate?

For context, each object in the array is a bundle definition for a specific use case. It's basically a list of imports that customers need to import to use a certain feature from our products.

We are setting up a dashboard to continuously track the bundle size for them across versions.